jaeger_client/codecs.py

```diff
@@ -34,6 +34,9 @@
 import six
 import urllib.parse
 
+SAMPLED_FLAG = 0x01
+DEBUG_FLAG = 0x02
```

Comment:
it's better to import these from `.span`

Reply:
I had tried that originally but `.span` already imports `codecs` so we end up with a circular import problem. Should I just move them to utils.py?

Reply:
constants.py would be better